modyn/supervisor/internal/triggers/drift/detector/alibi.py

```diff
@@ -42,18 +46,18 @@ def init_detector(self) -> None:
 
     def detect_drift(
         self,
-        embeddings_ref: pd.DataFrame | np.ndarray | torch.Tensor,
-        embeddings_cur: pd.DataFrame | np.ndarray | torch.Tensor,
+        embeddings_ref: pd.DataFrame | pd.Series | np.ndarray | torch.Tensor,
+        embeddings_cur: pd.DataFrame | pd.Series | np.ndarray | torch.Tensor,
```

Comment:
Why can the embeddings be a series now as well?

Reply:
not within Modyn, but in my offline exploration (iirc after PCA) I ended up with a series for detection. I didn't see any reason not to support this directly in the detector so I added support for it.